bolt-design-system/bolt
# Generate schema for new added icons

## Jira

http://vjira2:8080/browse/BDS-88

## Summary

Update bolt-icons component by giving a possibility to generate the updated schema in bolt-icon component.

## Details

Small changes in bolt-icons component and add `js-yaml` package to dependencies.

## How to test
Run this code locally. Add new svg icon to `packages/components/bolt-icons/src/svgs`. Run this code i command line in the project root: `npm run setup:full` then run this command: `npm start`. Run `docs-site` in the browser. Go to `components > Icon > Icon Name Variations` and check if new icon is added to it. Additionally, check if you have generated a file in: `www/build/data` called `icons.bolt.json`.
